framework/include/postprocessors/PseudoTimestep.h

```diff
@@ -0,0 +1,49 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+// MOOSE includes
+#include "GeneralPostprocessor.h"
+
+/**
+ * Computes a time step size based on user-specified CFL number
+ */
+class PseudoTimestep : public GeneralPostprocessor
+{
+public:
+  PseudoTimestep(const InputParameters & parameters);
+
+  virtual void execute() override;
+  virtual void initialize() override;
+  using Postprocessor::getValue;
+  virtual Real getValue() const override;
+
+protected:
+  const MooseEnum _method;
```

Comment:
I've used an enum class for this.  I think its a little weird because there is nothing to check that these enums are the same order as those used in the parms block.  But this is how it is usually done in MOOSE.  

```suggestion
  const enum class TimestepperMethod {
  SER,
  EXP,
  RDM
  }
```

Reply:
The axiom of choice! You suggested another style in another comment. We have 3 styles now. Which one should it be? I have no preference.